kinode/packages/app_store/app_store/src/lib.rs

```diff
@@ -73,12 +74,25 @@ fn init(our: Address) {
     loop {
         match await_message() {
             Err(send_error) => {
-                // for now, these are timer callbacks to already finished ft_workers.
-                print_to_terminal(1, &format!("got network error: {send_error}"));
+                // TODO: handle these more gracefully so only real send errors are printed
+                // // for now, these are timer callbacks to already finished ft_workers.
+                // print_to_terminal(1, &format!("got network error: {send_error}"));
+
+                let error_message = format!("got network error: {send_error}");
+                println!("{error_message}");
+                Response::new()
+                    .body(AppStoreResponse::SendError(error_message))
+                    .send()
+                    .unwrap();
             }
             Ok(message) => {
                 if let Err(e) = handle_message(&our, &mut state, &mut http_server, &message) {
-                    println!("error handling message: {:?}", e);
+                    let error_message = format!("error handling message: {e:?}");
+                    println!("{error_message}");
```

Comment:
are we sure we want to print with full vebosity here? I'll test with some basic interactions to see if it doesn't muddle up terminal too much

Reply:
Not sure about it. Previous behavior was verbosity 0, so just maintaining that. Happy to change.

Reply:
I set it it to verbosity 1 and removed the response, it's not really ever hit or listened to. Added some clarifications to the verbosity prints of chain and downloads so one knows where they come from.